src/components/Modal/Modal.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useRef } from 'react';
+import { useLockBodyScroll, useOnClickOutside } from 'utils/hooks';
+import * as S from './Modal.style';
+
+const Modal = ({ title, closeModal, children }) => {
+  const modalRef = useRef(null);
+  useLockBodyScroll();
+  useOnClickOutside(modalRef, closeModal);
+
+  const closeByESC = (e) => {
+    if (e.key === 'Escape') {
```

Comment:
```suggestion
    if (e.key === KEY.ESCAPE) {
```
상수로 분리해놓으면 좋을 것 같습니다